plugins/shortcuts/src/api/ShortcutApi.ts

```diff
@@ -28,6 +28,11 @@ export interface ShortcutApi {
    */
   shortcut$(): Observable<Shortcut[]>;
 
+  /**
+   * Returns an immediate snapshot of shortcuts, sorted by title
+   */
+  snapshot(): Shortcut[];
```

Comment:
Think we should probably call this `get` instead of `snapshot`. Think that it was probably a mistake that that old function was marked as private.